kolibri/plugins/user/assets/src/views/UserIndex.vue

```diff
@@ -37,12 +41,18 @@
     mixins: [commonCoreStrings],
     computed: {
       ...mapState(['pageName']),
+      ...mapGetters(['facilities']),
       immersiveProperties() {
         if (this.pageName === PageNames.SIGN_UP) {
           if (!this.$route.query.step) {
+            const backRoute =
```

Comment:
One fix for Navigation: 

If you are this point

<img width="422" alt="Screen Shot 2020-04-09 at 3 19 44 PM" src="https://user-images.githubusercontent.com/10248067/78945602-94892680-7a75-11ea-9793-ac0fd12301f2.png">

And you click "Create Account"

When you click the "X" exit link

Then you return to the SignIn page. (Currently it takes you to the signin/signup page).